macros/load_seed.sql

```diff
@@ -1,17 +1,33 @@
-{% macro load_seed(uri,pattern) %}
-{{ return(adapter.dispatch('load_seed')(uri,pattern)) }}
+{#
+    This macro includes options for compression, headers, and null markers.
+    Default options are set to FALSE. When set to TRUE, the appropriate
+    adapter-specific syntax will be used.
+
+    Argument examples:
+    compression=false
+    compression=true
+    headers=false
+    headers=true
+    null_marker=false
+    null_marker=true
+#}
+
+{% macro load_seed(uri,pattern,compression=false,headers=false,null_marker=false) %}
+{{ return(adapter.dispatch('load_seed', 'the_tuva_project')(uri,pattern,compression,headers,null_marker)) }}
```

Comment:
Had to add the_tuva_project name to the dispatch syntax so that the integration tests project can import and use the macro.

Reply:
I started playing with this locally - while the solution was simple, the dbt docs didn't exactly make it easy to find haha!

Reply:
@nrichards17 they really didn't! I stumbled across this when I was looking at how dbt_utils works.